Sandbox/app/src/main/java/novoda/com/sandbox/Application.java

```diff
@@ -0,0 +1,18 @@
+package novoda.com.sandbox;
+
+public class Application extends android.app.Application {
+
+    private static boolean signedIn = false;
```

Comment:
account manager needs to watch out, there's a new authentication pattern in town!